contracts/utilities/PausedTrueUSD.sol

```diff
@@ -0,0 +1,233 @@
+pragma solidity ^0.4.23;
+
+import "../HasOwner.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract PausedToken is HasOwner {
+    event Transfer(address indexed from, address indexed to, uint256 value);
+    event AllowanceSheetSet(address indexed sheet);
+    event BalanceSheetSet(address indexed sheet);
+
+    function totalSupply() public view returns (uint256) {
+        return totalSupply_;
+    }
+
+    function balanceOf(address _owner) public view returns (uint256 balance) {
+        return balances.balanceOf(_owner);
+    }
+
+    function allowance(address _owner, address _spender) public view returns (uint256) {
+        return allowances.allowanceOf(_owner, _spender);
+    }
+
+    function setAllowanceSheet(address _sheet) public onlyOwner returns(bool) {
+        allowances = AllowanceSheet(_sheet);
+        allowances.claimOwnership();
+        emit AllowanceSheetSet(_sheet);
+        return true;
+    }
+
+    function setBalanceSheet(address _sheet) public onlyOwner returns (bool) {
+        balances = BalanceSheet(_sheet);
+        balances.claimOwnership();
+        emit BalanceSheetSet(_sheet);
+        return true;
+    }
+
+    function transfer(address _to, uint256 _value) public returns (bool) {
+        revert("Token Paused");
+    }
+
+    function transferFrom(address _from, address _to, uint256 _value) public returns (bool) {
+        revert("Token Paused");
+    }
+
+    function burn(uint256 _value) public {
+        revert("Token Paused");
+    }
+
+    function mint(address _to, uint256 _value) public onlyOwner {
+        revert("Token Paused");
+    }
+    
+    function approve(address _spender, uint256 _value) public returns (bool) {
+        revert("Token Paused");
+    }
+
+    function _transferAllArgs(address _from, address _to, uint256 _value) internal {
+        revert("Token Paused");
+    }
+
+    function _transferFromAllArgs(address _from, address _to, uint256 _value, address _spender) internal {
+        revert("Token Paused");
+    }
+
+    function _approveAllArgs(address _spender, uint256 _value, address _tokenHolder) internal {
+        revert("Token Paused");
+    }
+
+    function increaseApproval(address _spender, uint _addedValue) public returns (bool) {
+        revert("Token Paused");
+    }
+    function decreaseApproval(address _spender, uint _subtractedValue) public returns (bool) {
+        revert("Token Paused");
+    }
+
+    function _increaseApprovalAllArgs(address _spender, uint256 _addedValue, address _tokenHolder) internal {
+        revert("Token Paused");
+    }
+
+    function _decreaseApprovalAllArgs(address _spender, uint256 _subtractedValue, address _tokenHolder) internal {
+        revert("Token Paused");
+    }
+}
+
+/** @title DelegateERC20
+Accept forwarding delegation calls from the old TrueUSD (V1) contract. THis way the all the ERC20
+functions in the old contract still works (except Burn). 
+*/
+contract DelegateERC20 is PausedToken {
```

Comment:
Probably just inherit the original

Reply:
the original DelegateERC20 inherits ModularStandardToken it's cleaner this way

Reply:
Can rename it then